boards/beagle/pocketbeagle_2/pocketbeagle_2_am6232_a53-pinctrl.dtsi

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2025 Ayush Singh, BeagleBoard.org Foundation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/dt-bindings/pinctrl/ti-k3-pinctrl.h>
+
+&pinctrl {
+	main_uart6_rx_default: main_uart6_rx_default {
+		pinmux = <K3_PINMUX(0x001c, PIN_INPUT, MUX_MODE_3)>;
```

Comment:
According to the AM62 datasheet, this pin is connected to pad A19. Address should be 0x00F4198.
```suggestion
		pinmux = <K3_PINMUX(0x0194, PIN_INPUT, MUX_MODE_3)>;
```